argus/backend/models/result.py

```diff
@@ -129,10 +129,10 @@ class ArgusGenericResultData(Model):
     status = columns.Ascii()
 
 class ArgusBestResultData(Model):
-    __table_name__ = "generic_result_best_v1"
+    __table_name__ = "generic_result_best_v2"
```

Comment:
This means we need to migrate current data ?

Reply:
teoretically yes, practicaly no - client was not updated and 'higher_is_better' param not set (thus no best result evaluation done).